core/src/luigi-config/features.js

```diff
@@ -0,0 +1,54 @@
+import { failFastFetch } from './navigation/queries';
+import { config } from './config';
+import jsonpath from 'jsonpath';
+
+const resolvers = {
+  apiGroup: (selector, data) =>
+    data.crds.some(crd => crd.includes(selector.apiGroup)),
+  configMapJsonPath: async (selector, data) => {
+    const { namespace, name, entry, jsonPath, expectedValue } = selector;
+
+    const cmUrl = `${config.backendAddress}/api/v1/namespaces/${namespace}/configmaps/${name}`;
+    const response = await failFastFetch(cmUrl, data.authData);
+    const configMap = await response.json();
+
+    const value = jsonpath.query(
+      JSON.parse(configMap.data[entry]),
```

Comment:
config map doesn't have to be JSON and it fails here if it's not

Reply:
also, this `jsonpath.query` might fail and we don't show why exactly, we should wrap this whole thing in a try/catch

Reply:
I've added a try/catch block with appropriate message, good point.

About the JSON... We assume here that the CM value will be a valid JSON, as we use JsonPath.